_posts/2019-01-03-dropping-puppet4.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: Dropping Puppet 4 support
+date: 2019-01-03
+github_username: bastelfreak
+twitter_username: BastelsBlog
+---
+
+Puppet 4 reached End of Life since 2018-12-31. There isn't an official support
```

Comment:
Not a native speaker, but I think it's "reached ... on" or "is ... since". This combination looks odd to me.

Reply:
ugh, I made many typos here. Fixed it.